client/components/site-title/index.jsx

```diff
@@ -54,9 +64,11 @@ class SiteTitleControl extends React.Component {
 						disabled={ disabled }
 						id="blogname"
 						onChange={ this.onChangeSiteTitle }
-						required
 						value={ blogname }
 					/>
+					{ isBlognameRequired && (
+						<FormInputValidation isError={ ! blogname } text={ translate( 'Required field.' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Required Fields' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).